Libraries/Text/TextInput/Multiline/RCTUITextView.m

```diff
@@ -27,7 +27,11 @@ @implementation RCTUITextView {
 
 static UIFont *defaultPlaceholderFont()
 {
+#if !TARGET_OS_OSX // [macOS]
   return [UIFont systemFontOfSize:17];
+#else // [macOS
+  return [UIFont systemFontOfSize:[NSFont systemFontSize]];
```

Comment:
Since we have an ifdef now, can we just change this to `[**NSFont** systemFontOfSize:...]`